services/actions/notifier_helper.go

```diff
@@ -103,45 +105,135 @@ func (input *notifyInput) Notify(ctx context.Context) {
 	}
 }
 
-func notify(ctx context.Context, input *notifyInput) error {
-	if input.Doer.IsActions() {
-		// avoiding triggering cyclically, for example:
-		// a comment of an issue will trigger the runner to add a new comment as reply,
-		// and the new comment will trigger the runner again.
-		log.Debug("ignore executing %v for event %v whose doer is %v", getMethod(ctx), input.Event, input.Doer.Name)
+func CreateScheduleTask(ctx context.Context, cron *actions_model.ActionSchedule, spec string) (int, error) {
```

Comment:
Could you please add the new functions to the end of this file or a new file? I would appreciate it.

If you see the diff page, you can find that it's very difficult to review.

Reply:
I noticed that there's a better place to go: `schedule.go`